pyproject.toml

```
@@ -70,3 +70,7 @@ ignore = [
     # wildcard imports significantly shorten test code,
     'F405',
 ]
+
+[tool.pytest.ini_options]
+python_files = ["test_*.py", "test_integration/__init__.py", "test.py"]
+norecursedirs = ["docker_files"] # otherwise we execute files which are intended to be executed inside a docker container
```

```suggestion
norecursedirs = ["contrib/p4/docker_files"]
```
I did not even know that this directory exists, so I was confused what this is referring to even with the comment.